tests/integration/modules/test_vmc_sddc_host.py

```diff
@@ -0,0 +1,87 @@
+"""
+    Integration Tests for vmc_sddc_host execution module
+"""
+import pytest
+import requests
+from saltext.vmware.utils import vmc_request
+
+
+@pytest.fixture
+def common_data(vmc_connect):
+    data = vmc_connect.copy()
+    data.pop("vcenter_hostname")
+    return data
+
+
+@pytest.fixture
+def request_headers(common_data):
+    return vmc_request.get_headers(common_data["refresh_key"], common_data["authorization_host"])
+
+
+@pytest.fixture
+def sddc_host_list_url(common_data):
+    url = "https://{hostname}/vmc/api/orgs/{org_id}/sddcs/{sddc_id}"
+    api_url = url.format(
+        hostname=common_data["hostname"],
+        org_id=common_data["org_id"],
+        sddc_id=common_data["sddc_id"],
+    )
+    return api_url
+
+
+@pytest.fixture
+def list_sddc_hosts(common_data, sddc_host_list_url, request_headers):
+    session = requests.Session()
+    response = session.get(
+        url=sddc_host_list_url,
+        verify=common_data["cert"] if common_data["verify_ssl"] else False,
+        headers=request_headers,
+    )
+    response.raise_for_status()
+    return response.json()
+
+
+@pytest.fixture
+def get_primary_cluster_id(common_data, salt_call_cli):
+    ret = salt_call_cli.run("vmc_sddc_clusters.get_primary", **common_data)
+    result_as_json = ret.json
+    return result_as_json["cluster_id"]
```

Comment:
**suggestion** this returned None for me, using the SALT-SDDC-TEST-NEW SDDC. If I should be using a different one for testing, please provide the name

Reply:
@waynew, You have to use "Prerna-Multi-host-SDDC" 
SDDC ID - d088f189-d3a3-43de-92b3-6c4ac26f9ac2
this is the same one i shared with you on slack channel.

Reply:
@waynew, please use new SDDC -  Salt-Multihost-SDDC-reshma (e5e387a1-216b-4002-a549-59b24126af5a) to run the tests.